### DISCUSSION TITLE
How to fix DTLS profile is not supported

### DISCUSSION THREAD

**Discussion by dovantuan02** [paullouisageneau/libdatachannel#1238] [General]
hi everyone. I have bug with DTLS when it run in armv7l. I was crosscompile and use openssl 3.0.2 (https://github.com/openssl/openssl.git) . 
i build openssl with target (linux-generic32)
Please help me
![image](https://github.com/user-attachments/assets/45131ccf-0f42-4a62-b238-d27ed810bded)

**Comment by paullouisageneau** [comment]
It looks like the libsrtp submodule was not built correctly and doesn't have GCM support.

By any chance, did you manually force `-DENABLE_OPENSSL=0` while building with OpenSSL? If so, remove the option.
